Realm.Shared/Realm.cs

```diff
@@ -676,80 +709,91 @@ public RealmResults<dynamic> All(string className)
         {
             RealmObject.Metadata metadata;
             if (!Metadata.TryGetValue(className, out metadata))
+            {
                 throw new ArgumentException($"The class {className} is not in the limited set of classes for this realm");
+            }
 
             return new RealmResults<dynamic>(this, metadata, true);
         }
 
-
         #region Quick ObjectForPrimaryKey
 
         /// <summary>
         /// Fast lookup of an object from a class which has a PrimaryKey property.
         /// </summary>
         /// <typeparam name="T">The Type T must be a RealmObject.</typeparam>
-        /// <param name="id">Id to be matched exactly, same as an == search. Int64 argument works for all integer properties supported as PrimaryKey.</param>
+        /// <param name="id">Id to be matched exactly, same as an == search. long argument works for all integer properties supported as PrimaryKey.</param>
```

Comment:
I wanted to use `long` since that would definitely be more familiar to c# developers than `Int64`. How about `An argument of type <c>long</c> works for all integer properties, supported as PrimaryKey.`